rules/prefer-to-contain.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const getDocsUrl = require('./util').getDocsUrl;
+const expectCase = require('./util').expectCase;
+const expectNotCase = require('./util').expectNotCase;
+const expectResolveCase = require('./util').expectResolveCase;
+const expectRejectCase = require('./util').expectRejectCase;
+const method = require('./util').method;
+const argument = require('./util').argument;
+
+module.exports = {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          !(
+            expectNotCase(node) ||
+            expectResolveCase(node) ||
+            expectRejectCase(node)
+          ) &&
+          expectCase(node) &&
+          (method(node).name === 'toBe' || method(node).name === 'toEqual') &&
+          (node.parent.parent.arguments &&
+            node.parent.parent.arguments.length === 1) &&
+          (argument(node).value === true || argument(node).value === false) &&
+          node.arguments[0] &&
+          node.arguments[0].callee &&
+          node.arguments[0].callee.property &&
+          node.arguments[0].callee.property.name === 'includes' &&
+          node.arguments[0].arguments.length === 1 &&
+          node.arguments[0].arguments[0]
+        ) {
+          const sourceCode = context.getSourceCode();
```

Comment:
could we move all these variables into the the `fix` closure to avoid the hit of calculating them unless people call `fix`?

Reply:
Moved